src/output.rs

```diff
@@ -420,10 +420,10 @@ pub fn newfmt_envelope(
     if crate::OUTPUT_FORMAT != 0 {
         meta.push_u(METADATA_OBJECT_FORMAT, crate::OUTPUT_FORMAT)
     }
-    if token != "" {
+    if !token.is_empty() {
```

Comment:
As a general remark, I think Clippy is just plain wrong here: this is not an improvement.  Maybe if there were an is_not_empty() method we could use, but not when the condition is being split up like this.

Reply:
Would you prefer me to un-stage these kind of changes from this PR?